src/assets/netteForms.js

```diff
@@ -27,17 +33,27 @@
 
 var Nette = {};
 
+function getHandler(callback) {
+	return function(evt) {
+		callback.call(this, evt || window.event);
```

Comment:
Ah, true; didn't know IE didn't support `evt.target` until version 9.. Can be fixed at the location you linked though, if you replace `var target = e ? e.target : event.srcElement;` with `var target = e.target || e.srcElement;`.